waku/v2/api/filter/filter.go

```diff
@@ -181,6 +187,10 @@ func (apiSub *Sub) resubscribe(failedPeer peer.ID) {
 	apiSub.multiplex(subs)
 }
 
+func possibleRecursiveError(err error) bool {
+	return strings.Contains(err.Error(), utils.ErrNoPeersAvailable.Error()) || strings.Contains(err.Error(), "dial backoff")
```

Comment:
I wonder if this would work so it's not necessary to check for the error string:
```
return errors.Is(err, utils.ErrNoPeersAvailable) || errors.Is(err, swarm.ErrDialBackoff)
```

Reply:
ah, thanks..that would definitely be better...will make the change.